takwimu/models/dashboard.py

```diff
@@ -340,6 +340,8 @@ class Meta:
 class IndicatorBlock(blocks.StructBlock):
     title = blocks.CharBlock()
     widget = IndicatorWidgetsBlock(min_num=1, max_num=1)
+    summary = blocks.RichTextBlock(required=False,
+                                   default='<p>Lorem ipsum dolor sit amet, adipiscing elitauris con lorem ipsum dolor sit amet.</p>')
```

Comment:
Maybe instead of `lorem ipsum`, default should be empty to be safe.

Reply:
Makes sense.